dynamic_programming/max_square.py

```diff
@@ -0,0 +1,68 @@
+"""
+Problem :
+Given a matrix of either 0 or 1, find the aera of the largest square composed of 1.
+"""
+
+
+from __future__ import annotations
+
+
+def max_square(matrix: list[list[int]]) -> int:
+    """
+    Return the aera of the largest square within matrix.
```

Comment:
`area`

Reply:
Yes of course, thanks to have figured out this typo !